packages/ihe-gateway-sdk/src/client/ihe-gateway.ts

```diff
@@ -11,7 +11,8 @@ export class IHEGateway {
    * @param xcpdGateways The OIDs and XCPD ITI-55 URLs of each organization to make a request to.
    * @param principalCareProviderNPIs The list of NPIs of the practitioners associated with the patient.
    * @param requestId Optional. Unique ID for the request. If not provided, one will be created.
-   * @returns an XCPD request to be used with an IHE Gateway.
+   *
+   * @returns an XCPD request to be used with an IHE Gateway. Throws Zod validation errors when if Organization OIDs or principalCareProviderNPIs don't meet their respective criteria.
```

Comment:
There's a particular format to indicate a function throws errors: https://tsdoc.org/pages/tags/throws/